tests/Phinx/Db/Adapter/SQLiteAdapterTest.php

```diff
@@ -1898,4 +1898,134 @@ public function provideTablesForTruncation()
             ['create table t(id integer primary key autoincrement)', 'T', 't'],
         ];
     }
+
+    public function testForeignKeyReferenceCorrectAfterRenameColumn()
+    {
+        $refTable = new \Phinx\Db\Table('ref_table', [], $this->adapter);
```

Comment:
Can you import class?

Reply:
Thit test suite contains many usages of that class, so i've maked new tests similar to existed. Also there are namespace \Phinx\Db\Table and class with exact "path". I propose to leave it as is :)

Reply:
Yep, this should be done using a code sniffer as automated approach instead.